tools/send-to-caas/send-utils.js

```diff
@@ -416,7 +416,7 @@ const props = {
   cardimage: () => getCardImageUrl(),
   cardimagealttext: (s) => s || getCardImageAltText(),
   contentid: (_, options) => {
-    const floodGateColor = getMetadata('floodGateColor') || '';
+    const floodGateColor = getMetadata('floodgatecolor') || '';
```

Comment:
Do we need to think about potential regression, i.e. existing metadata on pages using the camel-case variant `floodGateColor`? Maybe it makes sense to make the `getMetadata` function more robust in that it ignores case altogether and always does a lowercase before accessing properties from DOM elements.

Reply:
@honstar Thanks for looking, the property floodgatecolor is still not in use by consumers, is part of the feature work happening now. 
I think it’s a good idea to make the getmetadata function to ignore case for all properties, however other metadata properties are in use by consumers and can have potential impact, hence do you think it will be better to leave the generic function in current state.